git_manager/management/commands/init_bot_fork.py

```diff
@@ -0,0 +1,28 @@
+import os
+import sys
+import subprocess
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        for key in [ "BOT_REPO", "BOT_LOGIN", "BOT_TOKEN" ]:
+            assert hasattr(settings, key), \
+                   "You need to specify the {} variable in your settings before using this command".format(key)
+
+        git_path = os.path.join(settings.BASE_DIR, ".git_content_botfork")
+
+        if os.path.exists(git_path):
+            subprocess.check_call("rm -rf {}".format(git_path), shell=True)
+
+        github_repo = "https://github.com/" + settings.BOT_REPO
+        subprocess.check_call("git clone %s %s" % (github_repo, git_path), shell=True)
```

Comment:
again, this lib could make the code cleaner, but I'm not sure it's worth it https://gitpython.readthedocs.io/en/stable/

Reply:
Hm yea indeed ... I gave it a try though, and turns out it's not trivial to port the code because I manipulate the stdin to produce the diff and apply it later, without actually creating a file ...

An alternative would be to create the patched-branch right away by editing the file, but that imply quite a lot of changes in the code :/ Doesn't feel like it's worth it...

Reply:
Ok, let's forget that for now then, having this working seems more interesting :)